lib/Elevate/Leapp.pm

```diff
@@ -87,6 +96,64 @@ sub upgrade ($self) {
     return;
 }
 
+sub search_report_file_for_blockers ($self) {
+
+    my @blockers;
+    my @ignored_blockers = qw(
+      check_installed_devel_kernels
+      verify_check_results
+    );
+    my $leapp_json_report = LEAPP_REPORT_JSON;
+
+    if ( !-e $leapp_json_report ) {
+        ERROR("Leapp did not generated the expected report file: $leapp_json_report");
+        return [];
+    }
+
+    my $report = eval { Cpanel::JSON::LoadFile($leapp_json_report) } // {};
+    if ( my $exception = $@ ) {
+        ERROR("Unable to parse leapp report file ($leapp_json_report): $exception");
```

Comment:
should this be a blocker if we can't parse it?

Reply:
maybe return ['unable to parse leapp file thingy?']

Reply:
I'm not sure if that should be a blocker or not.  Maybe it should.  If running "leapp preupgrade" generated an invalid report file, then something is very wrong.  If it can't run "leapp preupgrade" properly, then I wouldn't trust it to run "leapp upgrade".